soh/soh/Enhancements/sfx-editor/SfxEditor.cpp

```diff
@@ -94,11 +94,6 @@ std::map<u16, std::tuple<std::string, std::string, SeqType>> sfxEditorSequenceMa
     {NA_BGM_UNDERGROUND,           {"Ganon's Castle Under Ground",         "NA_BGM_UNDERGROUND",             SEQ_BGM_WORLD}},
     {NA_BGM_GANONDORF_BOSS,        {"Ganondorf Battle",                    "NA_BGM_GANONDORF_BOSS",          SEQ_BGM_BATTLE}},
     {NA_BGM_GANON_BOSS,            {"Ganon Battle",                        "NA_BGM_GANON_BOSS",              SEQ_BGM_BATTLE}},
-    {NA_BGM_END_DEMO,              {"Seal of Six Sages",                   "NA_BGM_END_DEMO",                SEQ_BGM_WORLD}},
-    {NA_BGM_STAFF_1,               {"End Credits I",                       "NA_BGM_STAFF_1",                 SEQ_BGM_WORLD}},
-    {NA_BGM_STAFF_2,               {"End Credits II",                      "NA_BGM_STAFF_2",                 SEQ_NOSHUFFLE}},
```

Comment:
Would there be any difference in removing them and just making them all `SEQ_NOSHUFFLE` category? I think if we just changed the category it would leave this part of the map and up 1:1 with the vanilla sequences list

Reply:
I suppose there wouldn't be any difference but if that's the case we should put back the hyrule field tracks that have been removed. Also if we aren't allowing the sfx editor to shuffle those tracks then is there any point in them being in the list at all? I think it might actually simplify things a lot for them to not be there. Unless we plan on the SfxEditor doing other things that will be allowed for those sequences?

Reply:
I'm open to whichever you'd rather do here, but if I'm adding these back I'd also like to readd the Hyrule Field tracks and set them to noshuffle as well.

Reply:
is it possible to exclude them from the shuffle pool, but still allow the vinalla IDs to be overriden? My two cents would be to leave them, but exclude them from the pool. I believe @briaguya-ai mentioned [this as well](https://github.com/HarbourMasters/Shipwright/issues/1872#issuecomment-1345673955). If its not possible yet, then mark as noshuffle so we don't forget about them later.

Reply:
Making it appear in the editor so you can pick a new track but not available as a random selection or selectable track in the dropdown will require some new logic. I don't think it'll be too bad but should probably count as a new feature and we're in feature freeze right now. I can do the `SEQ_NOSHUFFLE` thing right away though so I will do that.